docs/community/community-call.md

```diff
@@ -0,0 +1,52 @@
+---
+id: community-call
+title: Update Call for SAP Community
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Community Call
+description: SAP Cloud SDK hosted community calls to share updates and demonstrate best practices for developers.
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+image:
+---
+
+## Community Call
+
+https://community.sap.com/resources/community-calls
+
+## Topics Backlog
+
+* Build an application usind SAP Cloud SDK in SAP Bussiness Application Studio
```

Comment:
Nice

Reply:
Do you think we can share our cloudsdk@sap.com email to get ideas about new topics from customers and share it on the future calls? Or maybe even your email?

Reply:
Also maybe move it behind the upcoming calls? Put it as a call to action.